src/Microsoft.SqlTools.ServiceLayer/DisasterRecovery/RestoreOperation/RestoreDatabaseHelper.cs

```diff
@@ -32,20 +32,38 @@ public class RestoreDatabaseHelper
         public RestoreConfigInfoResponse CreateConfigInfoResponse(RestoreConfigInfoRequestParams restoreConfigInfoRequest)
         {
             RestoreConfigInfoResponse response = new RestoreConfigInfoResponse();
-            RestoreDatabaseTaskDataObject restoreTaskObject = CreateRestoreForNewSession(restoreConfigInfoRequest.OwnerUri);
-            if (restoreTaskObject != null)
+            RestoreDatabaseTaskDataObject restoreTaskObject = null;
+            
+            try
             {
-                // Default Data folder path in the target server
-                response.ConfigInfo.Add(RestoreOptionsHelper.DataFileFolder, restoreTaskObject.DefaultDataFileFolder);
-                // Default log folder path in the target server
-                response.ConfigInfo.Add(RestoreOptionsHelper.LogFileFolder, restoreTaskObject.DefaultLogFileFolder);
-                // The db names with backup set
-                response.ConfigInfo.Add(RestoreOptionsHelper.SourceDatabaseNamesWithBackupSets, restoreTaskObject.GetDatabaseNamesWithBackupSets());
-                // Default backup folder path in the target server
-                response.ConfigInfo.Add(RestoreOptionsHelper.DefaultBackupFolder, restoreTaskObject.DefaultBackupFolder);
-            }
+                restoreTaskObject = CreateRestoreForNewSession(restoreConfigInfoRequest.OwnerUri);
+                if (restoreTaskObject != null)
+                {
+                    // Default Data folder path in the target server
+                    response.ConfigInfo.Add(RestoreOptionsHelper.DataFileFolder, restoreTaskObject.DefaultDataFileFolder);
+                    // Default log folder path in the target server
+                    response.ConfigInfo.Add(RestoreOptionsHelper.LogFileFolder, restoreTaskObject.DefaultLogFileFolder);
+                    // The db names with backup set
+                    response.ConfigInfo.Add(RestoreOptionsHelper.SourceDatabaseNamesWithBackupSets, restoreTaskObject.GetDatabaseNamesWithBackupSets());
+                    // Default backup folder path in the target server
+                    response.ConfigInfo.Add(RestoreOptionsHelper.DefaultBackupFolder, restoreTaskObject.DefaultBackupFolder);
+                }
 
+            }
+            catch(Exception ex) 
+            {
+                Logger.Write(LogLevel.Warning, $"Failed to create restore config info. error: { ex.Message}");
```

Comment:
In this case what will the response be set to? Should we track that it errored in some way?